src/test/java/org/opensearch/sdk/TestExtensionMapper.java

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+package org.opensearch.sdk;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+import org.opensearch.index.mapper.Mapper;
+import org.opensearch.index.mapper.MetadataFieldMapper;
+
+import java.util.Map;
+import java.util.function.Function;
+import java.util.function.Predicate;
+
+public class TestExtensionMapper {
```

Comment:
Nice test class, very thorough!   Can we either name it `TestMapperExtension` since that's the class we're testing, or just move some of these tests into the `TestExtensionInterfaces` class? I'm not sure we need all of them.